client/src/components/layout/PlayerTile.js

```diff
@@ -9,17 +9,19 @@ const PlayerTile = ({ player, selectedStats }) => {
 
   let playerStats = []
   selectedStats.forEach((stat) => {
+    let statObject = {name: stat}
     let stringedStat = player.stats[stat].toString()
     if (stat.includes("pct")) {
-      playerStats.push(stringedStat.substring(1))
+      statObject.value = stringedStat.substring(1)
+      playerStats.push(statObject)
     } else {
-      playerStats.push(stringedStat)
+      statObject.value = stringedStat
+      playerStats.push(statObject)
```

Comment:
consider moving this data manipulation to the backend before sending the data to be displayed on the front end